### DISCUSSION TITLE
Self-Review Questionnaire for Chair Candidates

### DISCUSSION THREAD

**Discussion by csarven** [solid/specification#568] [General]
While the general [role of the chair](https://www.w3.org/Guide/chair/role.html) sets key expectations, the following questions are intended to serve as a self-reflective exercise for both chair nominees and participants voting on [chair elections](https://www.w3.org/community/solid/charter/#chairs). Please add your own thoughts/considerations as well. We can perhaps add this questionnaire to the [Contributing Guide](https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md).

* Have you participated in CGs and/or WGs? Share your experience and contributions.
* Have you been an editor or author of a W3C Recommendation?
* When did you join the W3C Solid CG?
* What is your motivation for becoming chair? 
* What are some of your contributions to the W3C Solid CG (please include links)?
* How have you contributed to consensus in the W3C Solid CG?
* How have you contributed to promoting a healthy work environment?
* How have you promoted diversity and inclusion of voices from different backgrounds? 
* How have you helped prevent disruptive behaviour?
* How do you handle objections to proposals or decisions?
* What steps have you taken to ensure the group delivers on its commitments? Share your experience.
* Have you authored guidelines, processes, or policies in an open community? Share your experience.
* How do you navigate conflicts of interest (including personal or affiliation) while prioritising and making allowances for the community's interests and concerns?

**Comment by melvincarvalho** [comment]
Making a self-review questionnaire mandatory for chair candidates has several potential drawbacks:

### Diversity and Inclusion
1. **Bias against Underrepresented Groups**: Questions focusing on prior experience and contributions could unintentionally bias against candidates from underrepresented or marginalized communities who haven't had the same opportunities to contribute.

### Barriers to Entry
1. **Exclusivity**: Requiring answers to detailed questions about previous experience can create a barrier to entry for newcomers or those with less traditional backgrounds. 
2. **Overemphasis on Track Record**: While experience is important, the focus on past contributions may overshadow the potential for future impact. 
3. **Intimidation**: The comprehensive nature of the questions might be intimidating for some candidates, discouraging them from applying in the first place.

### Subjectivity and Interpretation
1. **Vague Criteria**: Questions like "How have you promoted diversity?" or "How do you handle objections?" are open to wide interpretation. 
2. **Self-Reporting Bias**: The self-reflective nature of the questionnaire could encourage exaggerated claims or subjective self-assessments that are hard to verify.

### Time and Resource Constraints
1. **Time-Consuming**: Filling out the questionnaire would require a significant time investment, which might discourage busy but otherwise excellent candidates.
2. **Resource-Intensive Review**: Reviewing the answers to these detailed questions could also be time-consuming for the selectors, potentially slowing down the appointment process.
  
### Objectivity and Transparency
1. **Lack of Metrics**: The questions are mostly qualitative, lacking objective metrics for assessment. 
2. **Ambiguity in Scoring**: Without a rubric or guidelines for evaluation, the answers could be assessed in an arbitrary manner, potentially leading to unfair outcomes.

For these reasons, while a self-review questionnaire could be a valuable tool for reflection and assessment, making it mandatory could have several unintended consequences. It may be more effective as an optional resource that candidates can choose to utilize.

**Reply by csarven** [reply]
The initial self-review questionnaire was meant to be a voluntary and self-reflective exercise, not a mandatory requirement.

This was all discussed in CG weekly meetings, and the group was asked to come up with improvements.

Let's work together to alleviate the concerns you are raising. I'm looking forward to hearing your suggested improvements.

**Reply by melvincarvalho** [reply]
Thank you for clarifying the voluntary nature of the self-review questionnaire. Although it being optional alleviates some concerns, there remain inherent issues that may need addressing to ensure fairness and inclusivity. Particularly, the focus on prior experience and contributions could potentially bias against candidates from underrepresented or non-traditional backgrounds, thus hindering diversity and inclusion. Furthermore, the comprehensive nature of the questionnaire might intimidate some individuals, potentially deterring a diverse range of candidates from participating. The efficacy and utility of such a questionnaire as a tool for evaluating chair candidates also remain questionable.

Additionally, it may be prudent to ascertain the level of interest within the community for the chair position before proceeding with a questionnaire. Understanding if there are more than three individuals interested in chairing the group could provide valuable context for evaluating the necessity and approach of the self-review questionnaire.

**Comment by timbl** [comment]
That questionnaire suggested by @csarven is one where @sarven checks most of the boxes himself.    It is a vision of a chair focussed on the incremental progress toward progress on new technical specs.  And a good list to be aware of.  But chairing the Solid Community group could also be about the community.

- Have you run hackathons?
- Have you run meetups?
- Have you coordinated meetups in different places?
- Have you made outreach to neighboring projects and gto them involved with this one?
- Have you helped  work on the overall roadmap of the movement and helped prioritize the work?
- Have you managed projects in software or content?
- Have you worked on public communications strategy for an org?
- Have you written public releases for a project?
- Have you fostered energy and participation in the global open source community?
- Have you coordinated crowdfunding around a project?
- Have you used a Solid pod to run a project?

and so on .. the reader can imagine other things.
The fact is that we need energy of man different types in the ecosystem.  If we had more than one chair, then that is easier to find a blend of energies.  Easier with three chairs rather than one. 

Participation in a community group, and being a chair especially, is an incredible gift to a project, and if the project is Solid, a gift to the world. Getting things going should be more about finding and inspiring and thanking people who can add energy and skills we need to make this fly, rather than building a  chair-filter to make sure chairs which don't meet a specific model don't get in.   My 2cents

**Reply by jeff-zucker** [reply]
@timbl wrote

> But chairing the Solid Community group could also be about the community.

I very much agree. But, as I understand it, the new CG charter explicitly marks as out of scope any outreach to or support of the community and leaves that to the Solid Team.   The things you mention like hackathons and fostering participation are not in the current CG scope.  If we want something that is about the community, that draws people and organizations in, that supports projects and facilitates onboarding of developers, it seems that there are only two  choices: 1) get back in and revise the CG charter or 2) move these activities under the Solid Team as proposed at https://github.com/solid/team/pull/39#issuecomment-1695990124.  In either case, I believe that if we have any hope of honoring the priority of constituencies, we need somethings that centrers and supports other constituencies than spec writers and offers them an opportunity to network similar to the CG weekly meetings but aimed specifically at the wider community.

**Reply by TallTed** [reply]
@timbl -- Please edit [your comment](https://github.com/solid/specification/discussions/568#discussioncomment-7241610), and change `where @sarven checks` to `where @csarven checks`, as the GitHub user `@sarven` is not otherwise connected to this discussion.

**Comment by ewingson** [comment]
{([triple-cross-posting this in the discussion-thread, in the chat, and in the public mailing list])}

I will do everything in the openness.
https://hackmd.io/pp76npqPSpKtyUb_pXkrCA?view

please allow the weekend to complete the motivation letter.
if anyone cares to read...
I try to be integrative, but there seem to be wounds that heal slowly (if at all).
please return to the table everyone.

my "more-than-two-cents"

**Comment by csarven** [comment]
I was suggested to share this here by multiple people. I originally hesitated because I did not want people to think or get confused that I'm interested in staying on or running for CG co-chair - I'm not.

---

I published an article titled [Self-Review: Chairing](https://csarven.ca/self-review-chairing) where I reflect on my experiences as the chair of the Solid CG (since 2019) and a snapshot of history. I am offering my experiences and insights as a valuable guidance for future contributors and chairs volunteering to take on this multidisciplinary challenge in order to achieve more democratic and diverse outcomes. This is one kind of a self-reflection, and hope that it is of some use to some.
